test/fixtures/block.ex

```diff
@@ -66,13 +66,13 @@ defmodule Fixtures.Block do
       fee_recipient: Random.execution_address(),
       state_root: Random.root(),
       receipts_root: Random.root(),
-      logs_bloom: Random.binary(256),
+      logs_bloom: :binary.bin_to_list(Random.binary(256)),
       prev_randao: Random.hash32(),
       block_number: Random.uint64(),
       gas_limit: Random.uint64(),
       gas_used: Random.uint64(),
       timestamp: Random.uint64(),
-      extra_data: Random.binary(30),
+      extra_data: :binary.bin_to_list(Random.binary(30)),
```

Comment:
not sure about this change, I think we want this to be a binary (a string) in elixir

Reply:
I think the problem with using only binaries is that it is not possible to control the size of that field.
For example, according to SSZ specs `logs_bloom` has to be a fixed vector with a default size of 256 bytes.
On the other hand, `extra_data` is a variable list (a byte list) with a maximum of 32 bytes.
I am not sure how it would be possible to make a difference between those types only using binaries :thinking:

Reply:
one possible solution would be to add these two extra functions in the library:
```defp decode_fixed_list(binary, {:int, 8}, size)```
```defp encode_fixed_size_list(binary, {:int, 8}, _size)```
That would keep the binary and it would not be necessary to change it to a list of bytes. Also, some sanitize_ssz would be not necessary anymore. 
What do you think @mpaulucci @Arkenan ?

Reply:
that same would apply to a vector of bytes like `logs_bloom`

Reply:
although, this solution breaks other types list `current_epoch_participation` that is indeed a list of u8 :sweat_smile: 
I think it would be necessary to add additional types to the schemas -> 
:byte_list and :vector_list

Reply:
Can't we just assume that `(binary, {:list, :uint8, size})` is a byte list (and the same for byte vectors)?

Reply:
so I think we should have `list of byte`, and `list of u8`. One is <<1, 2, 3 >> in Elixir, the other one [1, 2, 3]. They both end up being the same when serialized but the mapping is different when they unserialize. Those this make sense?

https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md#basic-types

Reply:
The size of the field is controlled by the `schema` definition, when they are an Elixir map both a vector and a list would of bytes would look like <<1, 2, 3>>.

Reply:
Ok, I think it is done now.